packages/table/src/table.tsx

```diff
@@ -316,8 +344,16 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
 
         const selectedRegions = (props.selectedRegions == null) ? [] as IRegion[] : props.selectedRegions;
 
+        let focusedCellCoordinates: IFocusedCellCoordinates = null;
+        if (props.focusedCellCoordinates != null) {
+            focusedCellCoordinates = props.focusedCellCoordinates;
+        } else if (props.allowFocus) {
+            focusedCellCoordinates = { col: 0, row: 0 };
```

Comment:
I wouldn't think it's possible to focus more than one cell. e.g. If you click + drag to select a region of cells, the focused cell would be the cell on which you clicked to initiate the drag.